### PR TITLE
API-5057: All dependencies upgraded to the latest version the app supports.

### PR DESCRIPTION
**Pull request by johanna-adhoc**
### Description

[API-5057 Upgrade DevPortal dependencies to wanted version](https://vajira.max.gov/browse/API-5057)
As a developer I want to update the application dependencies that are outdated to the wanted version available
so  the code uses the latest supported packages that the app can run without introducing breaking changes.

### Process
- [X] Tests added or updated for change

Updated test to comply with the eslint versions installed.
 
### Requested feedback

any feedback welcome

Screen Shot 2021-02-10 at 2 07 09 PM

### CODE REVIEW

**Review by anthony-kopczyk — COMMENTED** [review]
The files look great, but in my local when I run the unit test I get an error about an unmounted component in the SupportContactUsForm test that I'm not getting in the master branch. The test is still passing but is logging a warning. Looks like the issue stems from our Form.tsx component but I don't see the issue on other tests right now.

**Comment by johanna-adhoc** [comment]
> The files look great, but in my local when I run the unit test I get an error about an unmounted component in the SupportContactUsForm test that I'm not getting in the master branch. The test is still passing but is logging a warning. Looks like the issue stems from our Form.tsx component but I don't see the issue on other tests right now.

Good catch. That test unit was ending before the logic execution was done, I just added an extra await expect that will wait until all the logic runs before finishing the test case. Please recheck and let me know if the issue is gone on your side as well.

**Review by anthony-kopczyk — APPROVED** [review]
With the latest changes I'm not getting the warnings I was seeing before. Looks good to go!

**Review by mjlumetta — APPROVED** [review]
lgtm ✨

File: src/components/form/Form.test.tsx [hunk 1]
```diff
@@ -61,38 +65,24 @@ describe('Form', () => {
...
-      };
-    });
-    const onSubmitMock = jest.fn().mockImplementation(() => submitPromise);
-
-    const component = mount(<Form onSubmit={onSubmitMock} onSuccess={onSuccessMock} disabled={false} />);
+    const onSubmitMock = jest.fn().mockImplementation(
+      () =>
+        new Promise(resolve => {
+          setTimeout(() => {
+            resolve(new Error('promise trigger is set to default resolve'));
```
[the remark this answers is not in the document]
Comment (johanna-adhoc) [reply src/components/form/Form.test.tsx:72]: is just resolving the promise after a time period so the test can catch the button switching from submit to Sending... and then back to submit

**Review by TS000 — APPROVED** [review]
Latest changes fixed the issues I was having yesterday! Looks good!

**Pull request merged**